public/plugin.tsx

```diff
@@ -94,6 +102,40 @@ export class AssistantPlugin
       dataSourceManagement: setupDeps.dataSourceManagement,
     });
 
+    if (this.config.text2viz.enabled) {
+      setupDeps.visualizations.registerAlias({
+        name: 'text2viz',
+        aliasPath: '#/',
+        aliasApp: 'text2viz',
+        title: 'Natural language',
+        description: 'Generate visualization with a natural language question.',
```

Comment:
Shall we add i18n for these title and descriptions?

Reply:
Yes, missed that, I'll update